packages/keyring-controller/src/KeyringController.ts

```diff
@@ -10,7 +10,7 @@ import {
   signTypedData,
 } from '@metamask/eth-sig-util';
 import Wallet, { thirdparty as importers } from 'ethereumjs-wallet';
-import Keyring from 'eth-keyring-controller';
+import { KeyringController as EthKeyringController } from '@metamask/eth-keyring-controller';
```

Comment:
called this `EthKeyringController` to avoid namespace collision/confusion with the `Keyring` type defined below.

Reply:
This would close this issue: https://github.com/MetaMask/core/issues/774